src/libcompiler_builtins/lib.rs

```diff
@@ -180,7 +180,7 @@ pub mod reimpls {
             sr = sr.wrapping_add(1);
 
             // 1 <= sr <= u64::bits() - 1
```

Comment:
Yes, its consistent with the comment in the c implementation `/* 1 <= sr <= n_udword_bits - 1 */`.

Its types are defined [here](https://github.com/llvm-mirror/compiler-rt/blob/d89810afdc2862ea865f5c297ef4ea5839941d25/lib/builtins/int_types.h#L31) and [here](https://github.com/llvm-mirror/compiler-rt/blob/d89810afdc2862ea865f5c297ef4ea5839941d25/lib/builtins/int_types.h#L71).